docs/getting-started/archive-on-dockstore.rst

```diff
@@ -0,0 +1,68 @@
+Archiving an Entry on Dockstore
+===================================
+
+Tutorial Goals
+--------------
+
+-  Archiving an entry
+-  Unarchiving an entry
+
+This tutorial walks through the process of archiving entries from Dockstore.
+
+This action will make your entry read-only and less prominent in the Dockstore user interface.
+You can unarchive your entry back to normal at any time.
+
+
+Specific Changes When Archived
+-------------------------------
+
+After you archive the Dockstore workflow, it will be:
```

Comment:
Here you talk about archiving a workflow, then below, e.g., line 53, you talk about archiving a tool, workflow, notebook, or service. Either is good, but it should be consistent.

Reply:
> Here you talk about archiving a workflow, then below, e.g., line 53, you talk about archiving a tool, workflow, notebook, or service. Either is good, but it should be consistent.

IMHO, would be good if we use terminology that covers all entry types, because you can delete any type of entry.  "Tool, workflow, notebook, or service" would be ok, although it's long, and not scalable: it'll get even longer when we add the next Entry type.  Maybe it's time to reconsider our aversion to "entry" (or come up with a more palatable equivalent).

Reply:
Fair point in general, but in this case I was thinking of it as a tutorial, where you usually do one thing (and there is a screenshot of a workflow, or is a tool tool), so one option would have been at the beginning to say something like "this tutorial applies to workflows, tools, services, and notebooks, but here we'll be using a workflow as an example).

Nayeon, no need to change anything, just elaborating on my comment.